types/tampermonkey/index.d.ts

```diff
@@ -376,21 +420,149 @@ declare namespace Tampermonkey {
         version?: string;
     }
 
-    type ContentType = string | { type?: string | undefined; mimetype?: string | undefined };
+    type ContentType = string | { type?: string; mimetype?: string };
+
+    // GM_webRequest
```

Comment:
Should it be JSDoc comment? Can be useful

Reply:
I'm not sure what should be commented. Are you referring to the `ContentType` type?

Reply:
I meant something like this:
```
/**
* GM_webRequest
*/
interface WebRequestRuleParam
```
So user can know what `WebRequestRuleParam` is used for when importing type directly.
Feel free to skip if it makes little sense to you